tools/network/README.md

```diff
@@ -117,3 +117,61 @@ redirect: 0.000000
 starttransfer: 0.301009
 total: 2.322544
 ```
+## Speech-To-Text Curl Script: stt-curl.sh
+
+This bash script will issue periodic request to Watson Speech To Text (default period is every 2 seconds) and will print out cases where the response time exceeds a predefined threshold (default threshold is 2 seconds). The curl command generates all of the following:
+
+- **HTTP Response Headers** - needed to access transaction IDs
+- **JSON Response from STT** - which includes the STT ID (X-DP-Watson-Tran-ID and x-global-transaction-id)
+- **Time Elements** - which provide details about where time was spent during the transaction
+
+Note that this script first dumps all the results to a file (results.txt) and then uses awk to parse the last line of the file to get the total transaction time.
```

Comment:
I would make a note somewhere that before running the script you need to specify your apikey and if needed the correct Speech-To-Text hostname.